integration_tests/dbt_project.yml

```diff
@@ -11,7 +11,7 @@ clean-targets:         # directories to be removed by `dbt clean`
 vars:
   # shopify_using_fulfillment_event: true # set to true when regenerating docs
   # shopify_using_all_metafields: true # set to true when regenerating docs
-  shopify_schema: shopify_integration_tests_7
+  shopify_schema: shopify_integration_tests_8
```

Comment:
Can you also update the `+schema:` in the models config to the following:

```yml
models:
  +schema: "{{ 'shopify_sql_' ~ var('directed_schema','dev') if target.name == 'databricks-sql' else 'shopify_' ~  var('directed_schema','dev') }}"
```

Reply:
updated!